stanford_fellowship.features.inc

```diff
@@ -5,9 +5,28 @@
  */
 
 /**
+ * Implements hook_block_class_features_default_class().
+ */
+function stanford_fellowship_block_class_features_default_class() {
+  return array(
+    'ds_extras:fellowship_important_dates' => array(
+      'module' => 'ds_extras',
+      'delta' => 'fellowship_important_dates',
+      'css_classes' => 'pull-right span4  well',
```

Comment:
We don't export block classes in our features as they may change on a per site basis. We end up putting these into the  install profile. I can help with that once this PR is ready to be merged. Leave them in here for now as a reminder.